test/auth/features/oidc/broker.go

```diff
@@ -33,16 +38,20 @@ func BrokerSendEventWithOIDC() *feature.FeatureSet {
 	return &feature.FeatureSet{
 		Name: "Broker send events with OIDC support",
 		Features: []*feature.Feature{
-			BrokerSendEventWithOIDCTokenToSubscriber(),
-			BrokerSendEventWithOIDCTokenToReply(),
+			//BrokerSendEventWithOIDCTokenToSubscriberWithTLS(),
+			//BrokerSendEventWithOIDCTokenToReplyWithTLS(),
 			BrokerSendEventWithOIDCTokenToDLS(),
 		},
 	}
 }
 
-func BrokerSendEventWithOIDCTokenToSubscriber() *feature.Feature {
+func BrokerSendEventWithOIDCTokenToSubscriberWithTLS() *feature.Feature {
```

Comment:
Note: Change the test name to include the keyword `TLS` so that prow will enable TLS for this.